src/pages/PlayPage.js

```diff
@@ -11,23 +12,39 @@ export default function PlayPage({
   playing,
   duration,
 }) {
+  useShortcut(['a'], onAnswer)
+  useShortcut(['1'], onAnswer)
+  useShortcut(['b'], onAnswer)
+  useShortcut(['2'], onAnswer)
+  useShortcut(['c'], onAnswer)
+  useShortcut(['3'], onAnswer)
+  useShortcut(['s'], onPlay)
+
   return (
     <Container>
       {playing ? (
         duration && <Timer duration={duration} />
       ) : (
         <PlayButton onClick={onPlay}>&gt;</PlayButton>
       )}
-      {answers.map(answer => (
-        <Button
-          key={answer.title}
-          right={showAnswer ? answer.right : false}
-          wrong={showAnswer ? answer.wrong : false}
-          onClick={onAnswer}
-        >
-          {answer.title}
-        </Button>
-      ))}
+      <Hint tabIndex="0" aria-label="You can use this game by shortcuts">
+        <li>Use 's' to start the Song</li>
+        <li>Use first button with 'a' or '1'</li>
+        <li>Use second button with 'b' or '2'</li>
+        <li>Use third button with 'c' or '3'</li>
```

Comment:
Maybe you could say "Use first answer" instead of button. I think that´s a bit easier to understand.